internal/worker/client/command/query.go

```diff
@@ -0,0 +1,40 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/renatoaguimaraes/job-scheduler/internal/worker/proto"
+	"google.golang.org/grpc"
+)
+
+type QueryCommand struct {
+	client proto.WorkerServiceClient
+	args   []string
+}
+
+func NewQueryCommand(client proto.WorkerServiceClient) Runner {
+	return &QueryCommand{
+		client: client,
+	}
+}
+
+func (c *QueryCommand) Init(args []string) {
+	c.args = args
+}
+
+func (c *QueryCommand) Run() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Millisecond*100)
+	defer cancel()
+	command := proto.QueryRequest{
+		JobID: c.args[0],
```

Comment:
a length check is needed somewhere
right now, running `worker-client query` (without a 3rd argument) will panic